src/components/PageHeading.css

```diff
@@ -19,19 +19,6 @@
   margin-left: 70px;
 }
 
-.page-heading-title {
-  margin: 7px 0 5px 0;
-  line-break: auto;
-}
-
-.page-heading-description {
-  color: #777777;
-  font-style: italic;
-  font-weight: normal;
-  font-size: 19px;
-  line-height: 25px;
-}
-
 .page-heading-description .consent-data {
```

Comment:
You might want to double check if there are any `PageHeading`s in the code that still have a `consent-data` class as well, and verify if there's a change. This particular CSS block would never get executed now that `PageHeading` no longer uses the `page-heading-description` class.